---
layout: post
date: 2016-09-23
img: 

github:
doi: doi:10.1038/srep33640
article: http://www.nature.com/articles/srep33640
oa: 1
arxiv: 0
highlight: 1
software: 0

title: Schizophyllum commune has an extensive and functional alternative splicing repertoire
description:
 - >
  Recent genome-wide studies have demonstrated that fungi possess the machinery to alternatively splice pre-mRNA.
  However, there has not been a systematic categorization of the functional impact of alternative splicing in a fungus.
  We investigate alternative splicing and its functional consequences in the model mushroom forming fungus Schizophyllum commune.
  Alternative splicing was demonstrated for 2,285 out of 12,988 expressed genes, resulting in 20% additional transcripts.
  Intron retentions were the most common alternative splicing events, accounting for 33% of all splicing events, and 43% of the events in coding regions.
  On the other hand, exon skipping events were rare in coding regions (1%) but enriched in UTRs where they accounted for 57% of the events.
  Specific functional groups, including transcription factors, contained alternatively spliced genes.
  Alternatively spliced transcripts were regulated differently throughout development in 19% of the 2,285 alternatively spliced genes.
  Notably, 69% of alternatively spliced genes have predicted alternative functionality by loss or gain of functional domains, or by acquiring alternative subcellular locations.
  S. commune exhibits more alternative splicing than any other studied fungus.
  Taken together, alternative splicing increases the complexity of the S. commune proteome considerably and provides it with a rich repertoire of alternative functionality that is exploited dynamically.

short-title: Fungal alternative splicing
short-description: Alternative splicing in <i>S. commune</a>
tags:
 - Transcriptomics
 - Alternative splicing
 - Fungi
 - Sequence Analysis
---
